bin/xbps-install/transaction.c

```diff
@@ -228,6 +228,28 @@ show_transaction_sizes(struct transaction *trans, int cols)
 	return 0;
 }
 
+static bool
+all_pkgs_on_hold(struct transaction *trans) {
+	xbps_object_t obj;
+	const char *action;
+	bool all_on_hold;
```

Comment:
please `declare` and `initialize` in the same line, gets rid of extra useless lines.

Reply:
`bool all_on_hold = true`